app/models/shipit/commit.rb

```diff
@@ -289,20 +289,20 @@ def deploy_failed?
 
     def identify_pull_request
```

Comment:
However this should be renamed in `identify_merge_request `

Reply:
Resolved in: https://github.com/Shopify/shipit-engine/pull/1095/commits/ce8120a5f1e10b478c6b0facf5893a2027974d60